tests/pkg_libb2/main.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2018 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test the libb2 package
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ *
+ * @}
+ */
+
+#include <string.h>
+#include "embUnit.h"
+#include "blake2.h"
+#include "fmt.h"
+
+const char msg_hex[] = "000102030405060708090a0b0c";
+const char key_hex[] = "000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f";
```

Comment:
Why not directly declare and initialize this as an uint8_t array with the hex values?